thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/template.hbs

```diff
@@ -0,0 +1,306 @@
+<div class="manage-alert-explore">
+  {{#if (not isReplayPending)}}
+    {{!-- {{#if filteredAnomalies}} --}}
+      <div class="te-pill-selectors">
+        {{!-- Date range selector --}}
+        <div class="te-pill-selectors__label">Showing</div>
+        <ul class="te-alert-page-hzlist">
+          {{#each timeRangeOptions as |range|}}
+            <li class="te-alert-page-hzlist__item {{if range.isActive "te-alert-page-hzlist__item--active"}}" {{action "onRangeOptionClick" range}}>
+              {{range.name}}
+              {{#if (eq range.name "Custom")}}
+                : {{date-range-picker
+                  class="te-pill-selectors__range-picker"
+                  timePicker=false
+                  timePicker24Hour=true
+                  timePickerIncrement=timePickerIncrement
+                  maxDate=maxTime
+                  start=viewRegionStart
+                  end=viewRegionEnd
+                  ranges=predefinedRanges
+                  showCustomRangeLabel=false
+                  format=uiDateFormat
+                  serverFormat="YYYY-MM-DD HH:mm"
+                  applyAction=(action "onRangeSelection")
+                }}
+              {{/if}}
+            </li>
+          {{/each}}
+        </ul>
+      </div>
+
+      <div class="te-horizontal-cards te-content-block">
+        <h4 class="te-alert-page__subtitle">Alert Performance</h4>
+        <a class="te-pill-selectors__side-link" {{action "onClickTuneSensitivity" this}}>Customize sensitivity</a>
+        <div class="te-horizontal-cards__container">
+          {{!-- Alert anomaly stats cards --}}
+          {{#each anomalyStats as |card|}}
+            <ul class="te-horizontal-cards__card">
+              <li class="te-horizontal-cards__card-title">{{card.title}}</li>
+              <li class="te-horizontal-cards__card-value">{{card.value}}
+                <div class="te-horizontal-cards__card-tooltip">
+                  <span>
+                    <i class="glyphicon glyphicon-question-sign"></i>
+                    {{#tooltip-on-element}}{{card.text}}{{/tooltip-on-element}}
+                  </span>
+                </div>
+              </li>
+              <li class="te-horizontal-cards__card-text">
+                {{#if card.projected}}
+                  Projected: <strong>{{card.projected}}</strong>
```

Comment:
let's use a custom css class instead of `<strong>`. I believe the default styling for strong is too dark and too bold

Reply:
Good idea.